testing/citest/tests/openstack_smoke_test.py

```diff
@@ -0,0 +1,258 @@
+# Copyright 2017 Veritas Technologies, LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Integration test to see if the security group creation process is working for
```

Comment:
You only have security group here, but the test is presumably more general than that. If not call the test something like openstack_security_group_test.py